index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fp = require('./lib/fp.js');
+
+let main = module.exports = () => {
+  let list = fp.splice(process.argv, 2);
+  list = fp.map(list, (word) => word.toUpperCase());
+  let result = list.join(' ');
```

Comment:
don't forget to at some point print them out to the screen in their capital form.